whipper/common/program.py

```diff
@@ -470,6 +471,70 @@ def getHTOA(self):
         stop = track.getIndex(1).absolute - 1
         return start, stop
 
+    def getCoverArt(self, path, cover_art, release_id):
+        """
+        Get cover art image from Cover Art Archive if user chooses to
+
+        :type  cover_art: bool
+        :param cover_art: whether to fetch cover art or not
+        :type  path: str
+        :param path: where to store the fetched image
+        :type  release_id: str
+        :param release_id: a release id (self.program.metadata.mbid)
+
+        :rtype: str or None
+        """
+        if not cover_art:
+            return
+
+        import requests
+        import whipper
+
+        image_path = os.path.join(path, 'coverart.jpg')
+        user_agent = ("whipper/%s "
+                      "( https://github.com/whipper-team/whipper )" %
+                      whipper.__version__)
+        headers = {
+            'User-Agent': user_agent
+        }
+        url = 'https://coverartarchive.org/release/%r/front' % release_id
```

Comment:
Is there a reason you’re not using `musicbrainzngs`’s CAA calls but opting for calling the API directly yourself?

https://python-musicbrainzngs.readthedocs.io/en/v0.6/usage/#cover-art-data

Reply:
Yeah, i didn't know it exists :( .